content/docs/1.10.0/advanced-resources/os-distro-specific/talos-linux-support.md

```diff
@@ -103,14 +107,60 @@ If you were unable to include the `--preserve` option in the upgrade command, pe
 
 1. On the **Edit Node and Disks** page, add a disk and configure the following settings:
 
-    - **Path**: Specify `/var/lib/longhorn/`.
-    - **Storage Reserved**: Specify a value that matches your requirements. The default value is **30 Gi**. 
-    - **Scheduling**: Select **Enable**.
+   - **Path**: Specify `/var/lib/longhorn/`.
+   - **Storage Reserved**: Specify a value that matches your requirements. The default value is **30 Gi**.
+   - **Scheduling**: Select **Enable**.
 
 1. Click **Save**.
 
 Longhorn synchronizes the replicas based on the configured settings.
 
+### After v1.8.x
+
+The `--preserve` is no longer required, the flash if automatically set for `talosctl upgrade` command. [here](https://www.talos.dev/v1.8/introduction/what-is-new/#upgrades)
+
+## Talos Linux Version After to v1.10.x
+
+### Data Path Mounts
+
+Because Talos Linyux deprecated `.machine.disks` we recomend using `UserVolumeConfig` to mount a disk for Longhorn.[what's new in talos v1.10](https://www.talos.dev/v1.10/introduction/what-is-new/#user-volumes)
```

Comment:
```suggestion
Because Talos Linux deprecated `.machine.disks` we recommend using `UserVolumeConfig` to mount a disk for Longhorn.[What's new in talos v1.10](https://www.talos.dev/v1.10/introduction/what-is-new/#user-volumes)
```